applications/GeoMechanicsApplication/tests/cpp_tests/test_utilities/element_setup_utilities.h

```diff
@@ -52,6 +55,28 @@ class ElementSetupUtilities
     static Element::Pointer Create3D10NElement(const PointerVector<Node>& rNodes,
                                                const Properties::Pointer& rProperties);
     static Element::Pointer Create3D10NElement();
+
+    static Condition::Pointer Create2D3NLineCondition(const PointerVector<Node>& rNodes,
+                                                      const Properties::Pointer& rProperties);
+    static Condition::Pointer Create2D3NLineCondition();
+
+    template <class EntityPointerType>
+    static void AddVariablesToEntity(EntityPointerType& result,
```

Comment:
Just to comply with the Kratos Style Guide:
```suggestion
    static void AddVariablesToEntity(EntityPointerType& rpEntity,
```

Reply:
Done